src/store/mutations/mutations-model.ts

```diff
@@ -143,6 +143,10 @@ export const mutateCorrectionInformation = (state: StateIF, correctionInformatio
   state.stateModel.correctionInformation = correctionInformation
 }
 
+export const mutateCorrectionFilingDate = (state: StateIF, correctedFilingDate: string) => {
+  state.stateModel.correctionInformation.correctedFilingDate = correctedFilingDate
+}
+
```

Comment:
If someone could verify that this works locally for them before we merge, it was working for me this morning but now I changed something with my local machine and my all of my unit tests are broken and my setters are broken as well something with wrong with node_modules/http-signature/lib/signer.js:253

Reply:
I reinstalled my vscode and that seems to have fixed the issue, not sure what was wrong there.

Reply:
😅